Cargo.toml

```diff
@@ -12,7 +12,9 @@ exclude = ["assets/**/*", ".github/**/*"]
 
 [dependencies]
 bevy_proto_derive = { version = "0.1.0", path = "bevy_proto_derive" }
-bevy = "0.5"
+# bevy = "0.6"
+# bevy = { version = "0.6", default-features = false, path = "../bevy" }
```

Comment:
Yeah I do want to want to try and follow better practices and remove whatever features we don't need.